config/config.go

```diff
@@ -523,151 +516,11 @@ type FileSDConfig struct {
 	RefreshInterval model.Duration `yaml:"refresh_interval,omitempty"`
 }
 
-// RelabelAction is the action to be performed on relabeling.
```

Comment:
Moved to `pkg/relabel` and reduced `Relabel` prefix since package name is verbose enough.